### DISCUSSION TITLE
I am having loading issues at first time with my deployed version

### DISCUSSION THREAD

**Discussion by NicolasOmar** [anuraghazra/github-readme-stats#3585] [Q&A] [answered]
Hello, community,

Before starting, I would like to thank you for the work done on this impressive project.

Now, let me explain my situation step by step:
- I started using `github-readme-stats` around a year ago (checking my commits to be sure) without any issues until I deployed it in my personal vercel account a couple of months ago to avoid the loading errors you mentioned [in this article](https://github.com/anuraghazra/github-readme-stats/issues/1772).
- A time after that last change (around November), I started to see some loading issues when I entered my profile for the first time (with no cache or old cache data).
	- Those issues can be from showing the card or not showing it at all
	- After loading the same profile a second time, the issue disappears.

Before writing this discussion, I looked for a situation like mine here and in the issues section, but I couldn't find any related topic.
To continue the thread, please let me know what information you need to start solving this mystery hahaha.

Thank you for your time and effort for anyone who can give me a hand.
Nicolás.

Attached error screenshot:
![LoadingErrorScreenshot](https://github.com/anuraghazra/github-readme-stats/assets/16273464/e149c7a0-de06-4f48-8475-be8f30528dc4)

**Comment by qwerty541** [comment, marked as answer]
Hey, @NicolasOmar! Thanks for your report! This issue about card animation freezing was already discussed there https://github.com/anuraghazra/github-readme-stats/issues/3487. We came to conclusion that it is browser side issue which does not depend from our code. If you experience this too often then you're able to use `disable_animations=true` query option. I will attach an example below. If you have idea how we can deal with this issue, feel free to take part in discussion and share.

```html
<img src="https://github-readme-stats.vercel.app/api?username=anuraghazra&disable_animations=true" />
```

<img src="https://github-readme-stats.vercel.app/api?username=anuraghazra&disable_animations=true" />

**Reply by NicolasOmar** [reply]
Hey @qwerty541! Thank you for your answer.
I think this answer and solution should be enough to fix the problem (that is not happening as ofter for now), will add on my profile during today.
If by chance I got some alternative, I will revive this thread.

**Reply by qwerty541** [reply]
I am glad to help! Feel free to contact me if you need any further assistance.
